ps_wirepayment.php

```diff
@@ -442,4 +443,21 @@ public function getTemplateVarInfos()
             'bankwireCustomText' => $bankwireCustomText,
         );
     }
+
+    public function hookActionGetExtraMailTemplateVars($params)
+    {
+        if ($params['template']!='bankwire') {
+            return;
+        }
+
+        if (array_key_exists('{bankwire_owner}', $params['template_vars'])) {
```

Comment:
Shouldn't you check on `extra_template_vars` instead of `template_vars`? :thinking:

Reply:
Nope. This hook is called to add some params and it's implied that everybody who wanted already set the variables and the vars set to $params['template_vars'].
Of course there could be intercection with other module, that can set somthing other to those variables, but what if that other module will be called later that this one?